dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/AppPropertyConfigSource.java

```diff
@@ -32,21 +33,21 @@
 public class AppPropertyConfigSource extends InternalConfigSource implements DynamicConfigSource {
```

Comment:
You've added code to define the way that properties override each other if the same property is specified more than once.

It might be worth mentioning that in the class javadoc.

Could also include a sample of what the server.xml looks like.